plugins/core/gov.nasa.jpl.imce.oml.viewpoint/src/gov/nasa/jpl/imce/oml/viewpoint/AspectConceptClassDiagramService.xtend

```diff
@@ -0,0 +1,146 @@
+package gov.nasa.jpl.imce.oml.viewpoint
+
+import gov.nasa.jpl.imce.oml.model.terminologies.AspectSpecializationAxiom
+import gov.nasa.jpl.imce.oml.model.terminologies.Concept
+import gov.nasa.jpl.imce.oml.model.terminologies.ConceptSpecializationAxiom
+import gov.nasa.jpl.imce.oml.model.terminologies.Entity
+import gov.nasa.jpl.imce.oml.model.terminologies.EntityExistentialRestrictionAxiom
+import gov.nasa.jpl.imce.oml.model.terminologies.EntityRelationship
+import gov.nasa.jpl.imce.oml.model.terminologies.EntityRestrictionAxiom
+import gov.nasa.jpl.imce.oml.model.terminologies.ReifiedRelationship
+import gov.nasa.jpl.imce.oml.model.terminologies.SpecializationAxiom
+import java.util.HashSet
+import java.util.Set
+import org.eclipse.sirius.diagram.DDiagram
+import org.eclipse.sirius.diagram.DSemanticDiagram
+
+class AspectConceptClassDiagramService {

```

Comment:
Explain what this diagram is intended to show in terms of queries on the model:
- what are the boxes?
- what are the lines?